docs/src/rules/sort-imports.md

```diff
@@ -192,35 +192,61 @@ import {b, a, c} from 'foo.js'
 
 ### `ignoreCase`
 
-When `true` the rule ignores the case-sensitivity of the imports local name.
+When `false` (default), uppercase letters of the alphabet must always precede lowercase letters.
 
-Examples of **incorrect** code for this rule with the `{ "ignoreCase": true }` option:
+When `true`, the rule ignores the case-sensitivity of the imports local name.
+
+Examples of **incorrect** code for this rule with the default `{ "ignoreCase": false }` option:
 
 ::: incorrect
 
 ```js
-/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
-
-import B from 'foo.js';
+/*eslint sort-imports: ["error", { "ignoreCase": false }]*/
 import a from 'bar.js';
+import B from 'foo.js';
+import c from 'baz.js';
 ```
 
 :::
 
-Examples of **correct** code for this rule with the `{ "ignoreCase": true }` option:
+Examples of **correct** code for this rule with the default `{ "ignoreCase": false }` option:
 
 ::: correct
 
 ```js
-/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
-
-import a from 'foo.js';
+/*eslint sort-imports: ["error", { "ignoreCase": false }]*/
 import B from 'bar.js';
+import a from 'foo.js';
+import c from 'baz.js';
+```
+
+:::
+
+Examples of **correct** code for this rule with `{ "ignoreCase": true }` option:
+
+::: correct
+
+```js
+/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
+import a from 'bar.js';
+import B from 'foo.js';
 import c from 'baz.js';
 ```
 
 :::
 
+Examples of **incorrect** code for this rule with the `{ "ignoreCase": true }` option:
+
+::: incorrect
+
+```js
+/*eslint sort-imports: ["error", { "ignoreCase": true }]*/
+import B from 'foo.js';
+import a from 'bar.js';
+```
+
+:::
+
 Default is `false`.
```

Comment:
The default values are mentioned in the bottom of the option's section i feel this info should be on top with option's description can you shift them to the top for every option?

Reply:
I'm checking how other rules describe options but all rules have different formats 🤔. Anyway in this document, I'll address your comment.

---
1. `prefer-arrow-callback` - (allowUnboundThis)
They are telling default option as a sentence.
<img width="500" alt="스크린샷 2024-03-21 오후 3 37 04" src="https://github.com/eslint/eslint/assets/138835573/71cfe6b4-7321-42d8-9af7-29a8570b45df">

2. `no-unused-expressions` (allowTernary)
They doesn't talk about default option. It's just defined at the top of the document.
<img width="500" alt="스크린샷 2024-03-21 오후 3 36 50" src="https://github.com/eslint/eslint/assets/138835573/0081734b-6109-4b86-9f86-8d3eb18eb00e">

Reply:
Ya! but there is not a particular format for the description, only concept must be clear to the readers.